### DISCUSSION TITLE
An error has occurred. See the log file Null

### DISCUSSION THREAD

**Discussion by magicfinbow** [dbeaver/dbeaver#17229] [Q&A]
Hi folks, hope this is the right place to ask this.

I am trying to put DBeaver through RemoteApp (and CyberArk) and when the application is launched I get the error in the title.

No workspaces are created in my appdata - it's as if the application just doesn't even launch.  It launches perfectly when i RDP to the RDS server, I'm not sure how to get any more debugging to investigate.

**Comment by E1izabeth** [comment]
Maybe it's somehow related to this issue https://github.com/dbeaver/dbeaver/issues/17257
